static/js/reducers/ui_test.js

```diff
@@ -266,6 +270,48 @@ describe('ui reducers', () => {
   });
 
   describe('Enrollment', () => {
+    describe('enrollment message', () => {
+      it('should have a null default value', () => {
+        assert.equal(store.getState().ui.enrollMessage, null);
+      });
+
+      it('should let you set the message', () => {
+        return dispatchThen(setEnrollMessage("value"), [
+          SET_ENROLL_MESSAGE
+        ]).then(state => {
+          assert.equal(state.enrollMessage, "value");
+        });
+      });
+    });
+
+    describe('enrollment dialog error', () => {
+      it('should have a null default value', () => {
+        assert.equal(store.getState().ui.enrollDialogError, null);
+      });
+
+      it('should let you set the dialog error', () => {
+        return dispatchThen(setEnrollDialogError("value"), [
+          SET_ENROLL_DIALOG_ERROR
+        ]).then(state => {
+          assert.equal(state.enrollDialogError, "value");
+        });
+      });
+    });
+
+    describe('enrollment success/failure message', () => {
```

Comment:
the body of this describe block is exactly the same as the block at line 273 (apart from the 'it' output)